elements/lisk-api-client/src/transaction.ts

```diff
@@ -137,7 +134,10 @@ export class Transaction {
 				? codec.fromJSON(commandSchema, txInput.params as Record<string, unknown>)
 				: {},
 		};
-		if (authAccount.numberOfSignatures > 0) {
+		if (
+			authAccount.numberOfSignatures > 0 ||
+			(options?.multisignatureKeys && options?.includeSenderSignature)
```

Comment:
Not related to this particular issue, but we can remove `includeSenderSignature` because with the new protocol, sender signature is treated as any other signature

Reply:
Agree that we should remove the legacy logic, and agree to handle it as a separate issue:

https://github.com/LiskHQ/lisk-sdk/issues/7933